foundations/html_css/flexbox/flexbox-intro.md

```diff
@@ -26,30 +27,32 @@ Flexbox is a way to arrange items into rows or columns, where those items will f
 </p>
 <script async src="https://cpwebassets.codepen.io/assets/embed/ei.js"></script>
 
-We'll get into exactly what's going on here soon enough, but for now, uncomment the two flex related lines in the above CSS and check out the result. All 3 divs should now be arranged horizontally. If you resize the results frame with the "1x", ".5x" and ".25x" buttons you'll also see that the divs will 'flex'. They will fill the available area, and will each have equal width.
+We'll get into exactly what's going on here soon enough. But for now, uncomment the two flex related lines in the above CSS and check out the result. All 3 divs should now be arranged horizontally. If you resize the results frame with the "1x", ".5x" and ".25x" buttons you'll also see that the divs will 'flex'. They will fill the available area and will each have equal width.
 
-If you add another div to the HTML, inside of `.flex-container`, it will show up alongside the others, and everything will flex to make it fit.
+If you add another div to the HTML, inside of `.flex-container`, it will show up alongside the others, and everything will flex to fit within the available area.
 
 > #### Note
 > If it's hard to see what's going on in the small embedded CodePen, feel free to click the "Edit on CodePen" or "Fork on CodePen" button. This will bring the example into a full-sized environment. Some of the later examples might especially benefit from doing this.
```

Comment:
In regards to the comment above, would you mind making the following change?

```suggestion
**Note**: If it's hard to see what's going on in the small embedded CodePen, feel free to click the "Edit on CodePen" or "Fork on CodePen" button. This will bring the example into a full-sized environment. Some of the later examples might especially benefit from doing this.
```

Reply:
Sure thing. I had a query regarding this. Is this the only occurrence of such an instance where something like a heading: `>#### Some Heading` is wrapped inside a `<blockquote>` or `>` (for markdown).

I didn't find any information about styling something like this in the Layout Style Guide. Will have another look at it. :)

Reply:
No, there are are other instances of blockquotes in lessons that I think should be removed and turned into normal sub-sections (inline or not).

Reply:
I appreciate you letting me know that. Will be on the lookout for them. 

Shouldn't there be a gentle reminder in the Style Guide to discontinue that practice (`>#### Note`)? 
It seems that would speed up the process for modifying such instances whenever someone contributes.

Reply:
That might be more something I include in any issues regarding either heading fixes or for issues tailored to specific lessons (whereas right now the issues have encompassed several lessons).

That said, it could be worth making an issue for the Style Guide in the case of any future lessons if you think it'd be helpful down the line 🙂

Reply:
Yeah, let me ponder over it. Seems worthwhile to create an issue for the Style Guide. It would definitely help 🚀  anyone who comes across it.